Makefile

```diff
@@ -71,14 +71,20 @@ dev-pycontrails-bada:
 # QC, Test
 # -----------
 
-ruff: black-check
-	ruff check pycontrails tests
+lint:
+	ruff check pycontrails tests docs
+	ruff format --check pycontrails tests docs
 
-black:
-	black pycontrails tests
+format:
+	ruff format pycontrails tests docs
+
+ruff:
+	@echo -e "$(COLOR_YELLOW)The 'ruff' target is deprecated. Use 'make lint' instead.\n"
+	make lint
 
-black-check:
-	black pycontrails tests --check
+black:
+	@echo -e "$(COLOR_YELLOW)The 'black' target is deprecated. Use 'make format' instead.\n"
```

Comment:
Is there any reason to keep these? I think we are the only ones with workflows around them